src/lib/browse/InterventionLayer.svelte

```diff
@@ -127,3 +127,9 @@
     }}
   />
 </GeoJSON>
+
+<style>
+  :global(.border-popup .maplibregl-popup-content) {
```

Comment:
This is CSS selector syntax for maplibregl-popup-content contained inside of border-popup. Took me a minute to work this out. maplibre nests a few classes to draw the popup, I used chrome DOM inspector to figure it out